.github/workflows/add-release-reviewers.yml

```
@@ -9,29 +9,39 @@ permissions: {}
 
 jobs:
   add-reviewers:
-    if: ${{ github.event.label.name == 'release' }}
     # Tests that label is added on the PR
+    if: ${{ github.event.label.name == 'release' }}
     runs-on: ubuntu-latest
     permissions:
       contents: read
+      pull-requests: write
     steps:
       - id: get-members
-        uses: garnertb/get-team-members@0482f68c88601800f85145570ea93bf8fcfcf46f
-        with:
-          org: mui
-          team_slug: x
-          role: maintainer
-          token: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          DATA=$(gh api \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /orgs/${{ORG}}/teams/${{TEAM_SLUG}}/members \
+            | jq 'reduce inputs as $i (.; . += $i)') \
+          | echo "data=$DATA" >> $GITHUB_OUTPUT \
+          | echo "members=${{ join(fromJson(DATA).*.login) }}" >> $GITHUB_OUTPUT \
```

the `run: |` should allow you to run one command per line, removing the need to pipe everything. I think it makes it clearer to read, but feel free to ignore.

```suggestion
        run: |
          DATA=$(gh api \
            -H "Accept: application/vnd.github+json" \
            -H "X-GitHub-Api-Version: 2022-11-28" \
            /orgs/${{ORG}}/teams/${{TEAM_SLUG}}/members \
            | jq 'reduce inputs as $i (.; . += $i)')
          echo "data=$DATA" >> $GITHUB_OUTPUT
          echo "members=${{ join(fromJson(DATA).*.login) }}" >> $GITHUB_OUTPUT
```